plugins/sanity-plugin/README.md

```diff
@@ -0,0 +1,401 @@
+<a href="https://www.gatsbyjs.com">
+  <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
+</a>
+
+# Gatsby Starter Sanity Homepage
+
+Create a homepage using Gatsby and Sanity. This starter demonstrates how to use Sanity to build a homepage and can be customized to match your own visual branding.
+
+[View the Demo](https://gatsbysanityhomepage.gatsbyjs.io/)
+
+**Note:**
+This version of the Sanity homepage starter is written in JavaScript. If you want to use Sanity but TypeScript is more your style, there is also a TypeScript version maintained on [GitHub](https://github.com/gatsbyjs/gatsby-starter-sanity-homepage-ts).
+
+## Quick start
+
+You will need a Sanity account to use this starter. You can create a new account or use an existing one.
+
+1. **Create a Gatsby site**
+
+   Use the Gatsby CLI to get started locally:
+
+   ```sh repo
+   npx gatsby new my-homepage https://github.com/gatsbyjs/gatsby-starter-sanity-homepage
+   ```
+
+1. **Set up Sanity Studio**
+
+   1. In the `studio` directory, install dependencies for Sanity Studio:
+
+      ```sh
+      yarn
+      ```
+
+   1. Create a new Sanity project by running:
+
+      ```sh
+      yarn sanity-init
+      ```
+
+      This will prompt you to log in if this is your first time using Sanity CLI.
+
+   1. Deploy the Sanity GraphQL API for your new project:
+
+      ```sh
+      yarn deploy
+      ```
+
+   1. Optionally, to import the demo content for this starter run:
+
+      ```sh
+      yarn import
+      ```
```

Comment:
Got another conflict for keywords that `yarn` uses here.

![Screen Shot 2022-04-05 at 3 06 18 PM](https://user-images.githubusercontent.com/1790506/161830779-9210dbce-e3a6-4ac9-b65e-968c1f1703c8.png)

Reply:
Probably update this to `yarn run import` and call it good

Reply:
Renamed it to `yarn sanity-import` to avoid collisions